auth/auth_user.py

```diff
@@ -9,20 +9,19 @@
 
 
 class AuthUser:
-	phantom_set = None
 	user_profile = None # Cache the user
-
+	phantom_ids = []
 
 	@classmethod
-	def get_phantom_set(cls):
-		if cls.phantom_set:
-			return cls.phantom_set
-		cls.phantom_set = sublime.PhantomSet(PiecesSettings.output_panel, "pieces_auth")
-		return cls.phantom_set
+	def create_new_phantom(cls,html):
+		if cls.phantom_ids:
+			for phantom in cls.phantom_ids:
+				PiecesSettings.output_panel.erase_phantom_by_id(phantom)
+		cls.phantom_ids.append(PiecesSettings.output_panel.add_phantom("auth_phantom", 
+			sublime.Region(0, 0), html, sublime.LAYOUT_INLINE))
```

Comment:
we need to remove our previous panel everytime? and the previous panels before?

Reply:
yes we need to clear it.